nilearn/datasets/atlas.py

```diff
@@ -1484,12 +1628,25 @@ def fetch_atlas_pauli_2017(version='prob', data_dir=None, verbose=1):
 
     Returns
     -------
-    sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, contains:
 
-        - maps: 3D Nifti image, values are indices in the list of labels.
-        - labels: list of strings. Starts with 'Background'.
-        - description: a short description of the atlas and some references.
+            - 'maps': :obj:`str`, path to nifti file containing the
+              :class:`~nibabel.nifti1.Nifti1Image`. If ``version='prob'``,
+              the image shape is ``(193, 229, 193, 16)``. If ``version='det'``
+              the image shape is ``(198, 263, 212)``, and values are indices
+              in the list of labels (integers from 0 to 16).
+            - 'labels': :obj:`list` of :obj:`str`. List of region names. The
+              list contains 16 values for both probabilitic and deterministic
+              versions.
+
+                .. note::
+                    For the deterministic version, 'Background' is not
```

Comment:
isn't it for the probabilistic version? because there is no probabilistic map of the background

Reply:
I see, I understand now -- "background" is never included in the list of labels, and that is a mistake only in the deterministic version.

so the image values are actually not indices into the list of labels because they are offset by 1, right?

Reply:
>so the image values are actually not indices into the list of labels because they are offset by 1, right?

Yes, absolutely. Thanks for raising this. :+1: 
I've updated this part of the docs in 5521855 
Hopefully it is clearer like this.